### DISCUSSION TITLE
Translation of docs

### DISCUSSION THREAD

**Discussion by jorgebef** [amannn/next-intl#170] [Ideas]
Hello,
I was chatting about the i18n implementation with someone who is Spanish speaking and realised that the documentation is only in English.

It would be nice if the documentation for an i18n library is actually internationalized 😝
I can help with Spanish as I am native Spanish and English.
I can help with other languages, but I believe it best to leave it to a native speaker of those languages.

I don't know whether you are interested in going down that route now or whether you would rather wait until the `app` directory is out of beta?

**Comment by amannn** [comment]
Hey! Ha, that's a fair point indeed! 😄

Thank you so much for proposing to volunteer! If we add new languages, I'd certainly need help from native speakers as I only speak English and German unfortunately.

I think for the time being I'd propose to turn this into an issue and to see how many users are interested in this. I think the effort of supporting multiple languages is quite extensive, considering we'd have to translate all the existing content but also to keep it in sync in the future. I think currently this endeavour might be a bit too ambitious, given the current popularity of the library. But it might be something to look into in case the user base grows!

Out of curiosity: Does your colleague not speak English at all? What's his/her experience with other developer documentation? Is there much content available in Spanish?

I briefly tried what the next-intl docs look like if I use Google Translate to turn it into German and I didn't find it _too_ bad TBH. Of course it's not 100% correct, but still helping. Also code samples remain as-is, that's quite good.

**Reply by jorgebef** [reply]
Well, anyone in software dev kind of has to understand English to a certain level, but that doesn't take away the fact that they much prefer any content in their native language, and actively search for tutorials or explanations.

It is quite ambitious tbh, and I agree that making it an issue would be best for now.

Regarding popularity, I believe this repo has a lot of potential given the problem it solves and the fact that the solution provided by the nextjs dev team is more inclined on a basic approach and statically generated, whereas this library is quite a bit more flexible.

Hopefully we can gain some traction, brilliant minds and keep evolving!!

**Reply by amannn** [reply]
Thanks for providing more background, I can relate to that!

Sounds great, this is the issue I've created from this discussion btw.: https://github.com/amannn/next-intl/issues/171

> Hopefully we can gain some traction, brilliant minds and keep evolving!!

Yes! 🙌 ❤️
